benefits/settings.py

```diff
@@ -234,14 +234,23 @@ def RUNTIME_ENVIRONMENT():
 STATICFILES_DIRS = [os.path.join(BASE_DIR, "benefits", "static")]
 # use Manifest Static Files Storage by default
 STORAGES = {
+    "default": {
+        "BACKEND": "django.core.files.storage.FileSystemStorage",
+    },
     "staticfiles": {
         "BACKEND": os.environ.get(
             "DJANGO_STATICFILES_STORAGE", "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
         )
-    }
+    },
 }
 STATIC_ROOT = os.path.join(BASE_DIR, "static")
 
+#  User-uploaded files
+
+MEDIA_ROOT = os.path.join(STORAGE_DIR, "uploads/")
+
+MEDIA_URL = "/uploads/"
```

Comment:
Can we call these `media` instead of `uploads`? We can be a little obscure that there is a way to upload files into the site :wink: 

Sorry I know this is going to impact a bunch of other changes.